File: RemoveDuplicate2.py
# https://leetcode.com/problems/remove-duplicates-from-sorted-array-ii
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        l=2
        for i in range(2,len(nums)):
            if nums[i]!=nums[l-2]:
                nums[l]=nums[i]
                l+=1
        return l

File: RemoveDuplicate2.java
// https://leetcode.com/problems/remove-duplicates-from-sorted-array-ii
public class RemoveDuplicate2 {
    public int removeDuplicates(int[] nums) {
        int l=2;
        for(int i = 2; i<nums.length;i++){
            if(nums[i]!=nums[l-2]){
                nums[l]=nums[i];
                l++;
            }
        }
        return l;
    }
}
